packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/ResultOptionsBar/index.js

```diff
@@ -35,42 +35,48 @@ export class ResultOptionsBar extends Component {
     } = this.props
 
     return (
-      <Menu attached="top" style={{ position: 'relative' }} tabular>
-        {itemEnabled && (
+      <Menu
+        attached="top"
+        className="icon"
```

Comment:
remove className and replace with bool props `icon` and `secondary`

Reply:
secondary goes to Menu.Menu